src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useMemo } from "react";
+import "./RadioModal.scss";
+import { useSelector } from "hooks/useSelector";
+
+interface PropsType {
+  volume: number;
+  setVolume: Function;
+}
+
+export const RadioModal: React.FunctionComponent<PropsType> = ({
+  volume,
+  setVolume,
+}) => {
+  const { radioStations } = useSelector((state) => ({
+    radioStations: state.firestore.data.venues?.playa?.radioStations,
+  }));
+
+  const sound = useMemo(
+    () => (radioStations ? new Audio(radioStations[0]) : undefined),
```

Comment:
Should probably do a length check: `radioStations && radioStations.length ? new Audio(radioStations[0]) : undefined` ?

Reply:
Or `new Audio(undefined)` is OK then no prob.